### ISSUE TITLE
Windows executable is mistakenly detected as "malware"

### ISSUE THREAD

**Issue by skyem123** [vkbo/novelWriter#580]
I am not sure if this is a bug or not, but I am reporting it here just in case. Sorry if this is the wrong place.

* A clear and concise description of what the bug is.
Malwarebytes thinks the windows executable is malware and gets rid of it. It claims that it was detected by AI (Malware.AI.339339336 https://blog.malwarebytes.com/detections/malware-ai/)

* Steps to reproduce the behaviour
On windows, Install malwarebytes, then install novelWriter, then scan the install folder with malwarebytes. The executable should be detected as "malware".

* A clear and concise description of what you expected to happen.
For the executable to not be detected.

**Comment by vkbo** [comment]
Hi, thanks for the report.

Yes, this is a false positive on the pyinstaller tool that is used to package novelWriter for Windows. This happens all the time, and comes and goes with some virus scanners. I check all the packages with Virus Total before uploading them, and there's always one or two of the 50+ scanners that flag it. It varies which ones, so clearly it's just a matter of the scanners updating their detection algorithm. I'll try not to use the latest pyinstaller when I make these packages to avoid this issue as much as possible.

There are numerous threads on this on the pyinstaller repository:
https://github.com/pyinstaller/pyinstaller/issues?q=is%3Aissue+virus+is%3Aclosed

**Comment by vkbo** [comment]
I've tested with several different versions of pyinstaller, and a few scanners on virus total flag them. About 5-10%. There's a suggested solution here: https://stackoverflow.com/questions/43777106/program-made-with-pyinstaller-now-seen-as-a-trojan-horse-by-avg

I'll try this later, but I may try to find another way to distribute on Windows as this is annoying. A lot of projects are getting a lot of reports on this same thing, and I'm reconsidering providing the installer at all. It is pretty clear that these are false positives, but I also don't want to be flooded with issue reports about it.

**Comment by skyem123** [comment]
Oh... That sounds like a real pain. Thank you for looking into this. I got it working on my computer by installing it with pip, I guess that works since it doesn't use pyinstaller.

**Comment by vkbo** [comment]
I've managed to create a standalone folder that includes novelwriter, python, and the dependencies, and where the app can be launced from within the folder with `pythonw.exe novelWriter.pyw`. I've also managed to get the Inno Setup installer to install this and set up the launcher icons and file association. I think this can replace the pyinstaller-based install package. I'll provide both package alternatives for the next release so people can test it.
